platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -42,6 +42,11 @@ For Java, Groovy, Kotlin, and Android compatibility, see the [full compatibility
 
 <!-- Do not add breaking changes or deprecations here! Add them to the upgrade guide instead. -->
 
+### Bearer Token Authentication for Wrapper Download
+
+When downloading Gradle distributions from an HTTP/HTTPS backend (secure connection recommended) the Wrapper can now also use bearer token based authentication (in addition to the basic user-password method supported previously).
```

Comment:
```suggestion
When downloading Gradle distributions from an HTTP or HTTPS backend, a secure connection is highly recommended, the Wrapper now supports bearer token-based authentication. 
This is in addition to the basic username and password method supported in previous versions.
```

Reply:
Besides your suggestion, I've also added more content to the notes. Please take another look and feel free to trim it if it's too much.

Reply:
Also, a nitpick, I don't like how you rewrote the "When downloading Gradle ..." sentence. The secure connection thing already was the recommended way for authenticated connections, we didn't change that and it's not the focus of the message. But I've applied your version and I don't feel too strongly about it.